src/components/10-atoms/toggle-switch/index.js

```diff
@@ -47,6 +47,13 @@ class AXAToggleSwitch extends LitElement {
       </label>
     `;
   }
+
+  handleChange(event) {
```

Comment:
Why did you add that handler? (i think i know.... but want to read your explanation)

Reply:
> Why did you add that handler? (i think i know.... but want to read your explanation)

I added a handler to make sure that the `active` property is always updatet and reflected correctly.
+ I think I should also dispatch an event and not only call the callback function, but I'll need some help with this.

Reply:
You dont need to dispatch an event. It is already dispatched by the browser when the `checked` value changed at the input element.